Formula/airdrop-cli.rb

```diff
@@ -4,7 +4,7 @@ class AirdropCli < Formula
   url "https://github.com/vldmrkl/airdrop-cli/archive/1.0.1.tar.gz"
   sha256 "2e93796692fddb13e362bfd9c204bfc2f24ef22fadaf09efe97201d0bc9b54c3"
   license "MIT"
-  head "https://github.com/vldmrkl/airdrop-cli.git"
+  head "https://github.com/vldmrkl/airdrop-cli.git", branch: "main"
```

Comment:
Thanks for the PR, but I am confused why would it fix the problem since there is no `main` branch in this repository?